runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -546,6 +548,9 @@ public synchronized void invalidateLayout() {
 
                         log.debug("Layout server {} responded with layout {}", s, l);
                         return l;
+                    } catch (InterruptedException ie) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)